### PR TITLE
🎨 Added emojis to CI/CD

### PR DESCRIPTION
**Pull request by simon1hofmann**
## Description

Added emojis to every workflow.

## Checklist:

- [x] The pull request only contains commits that are related to it.
- [ ] I have added appropriate tests and documentation.
- [ ] I have added a changelog entry.
- [ ] I have created/adjusted the Python bindings for any new or updated functionality.
- [x] I have made sure that all CI jobs on GitHub pass.
- [x] The pull request introduces no new warnings and follows the project's style guidelines.

### CODE REVIEW

**Comment by simon1hofmann** [comment]
@marcelwa The chosen emojis are a mixture between MQT and what I thought fits best.

**Comment by marcelwa** [comment]
@simon1hofmann could we use 🪟 for Windows?

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Coverage CI
+name: 📈 • Coverage
```
Comment (Drewniok) [inline .github/workflows/coverage.yml:1]: Just to double-check: The ``CI`` was intentionally deleted, right?

Comment (simon1hofmann) [reply .github/workflows/coverage.yml:1]: Yes, I wanted to reserve `CI` for the actual macOS, windows, ubuntu and pyfiction CI's and use, e.g., 📈 • Coverage and 📝 • CodeQL for the others.

Comment (marcelwa) [reply .github/workflows/coverage.yml:1]: We're often using ☂️ for coverage due to Codecov's logo. Do you think that would work here, too?

Comment (simon1hofmann) [reply .github/workflows/coverage.yml:1]: Sure, sounds good!

**Review by marcelwa — APPROVED** [review]
Many thanks!

**Pull request merged**